lib/shopify_app/session/in_memory_session_store.rb

```diff
@@ -12,6 +12,10 @@ def self.store(session, *args)
       id
     end
 
+    def self.retrieve_by_jwt(payload)
```

Comment:
This looks vestigial.

Also does this class make sense / is it used anywhere anymore?

Reply:
Oh nvm I see that we subclass from it. Either way, I don't think it should have `retrieve_by_jwt` method anymore.